### DISCUSSION TITLE
API-Server in TI RU

### DISCUSSION THREAD

**Discussion by fue-medatixx** [Digitaler-Impfnachweis/certification-apis#22] [Q&A] [answered]
From the current API description:
url: https://api.ru.impfnachweis.info    description: Endpoint for medical practitioners with TI access only - STAGING api server

Is the server in the RU already online? What's his IP address?

**Comment by oliver-steinbrecher** [comment, marked as answer]
Routing and Endpoints are set up:
Active: 10.30.129.10/32
Passive: 10.30.129.75/32

The SMCB authentication will be provided until end of this week.

**Comment by oliver-steinbrecher** [comment]
Please note the following update : https://github.com/Digitaler-Impfnachweis/certification-apis/blob/master/SMCB-Authentication.md#official-endpoints
